sandbox/mock.go

```diff
@@ -89,9 +81,6 @@ func (m *MockSandbox) AnyRecentTransaction(txID tx.ID) bool {
 
 func (m *MockSandbox) Validator(addr crypto.Address) *validator.Validator {
 	val, _ := m.TestStore.Validator(addr)
-	if val == nil {
-		return m.TestSandboxVals[addr]
-	}
 	return val
```

Comment:
what if the validator was nil here? are we check that in the upper layers?

Reply:
It can be `nil`. In that case, the new validator should be created using `MakeNewValidator`. Please note that this is a mock and is used only for testing purposes.

Reply:
Ok, got it.